### PR TITLE
Improved ZH-Hans translate.

### PR DESCRIPTION
**Pull request by zrzjohn**
Improved Simplified Chinese translate. Make the translation more accurate.

- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)? 
I followed the contributing guidelines.
- [x] Have you explained what your changes do, and why they add value to the Guides?
Yes, I have.

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by Mr1262** [comment]
👍

**Comment by MikeMcQuaid** [comment]
Need to fix the failing CI tests on this PR.

**Comment by jor514** [comment]
gp

**Comment by moonhilal** [comment]
I did ,t understand anything

**Comment by iancanderson** [comment]
Hi @zrzjohn - please fix the errors in CI before requesting our review. For one thing, it looks like you auto-renamed `pr` to `Pull Request`, which resulted in some unintended changes, like renaming `best-practices` to `best-Pull Requestactices`

**Comment by zrzjohn** [comment]
> Hi @zrzjohn - we can review this after you fix the errors in CI. For one thing, it looks like you auto-renamed `pr` to `Pull Request`, which resulted in some unintended changes, like renaming `best-practices` to `best-Pull Requestactices`

I did understand anything.

**Comment by zrzjohn** [comment]
> > Hi @zrzjohn - we can review this after you fix the errors in CI. For one thing, it looks like you auto-renamed `pr` to `Pull Request`, which resulted in some unintended changes, like renaming `best-practices` to `best-Pull Requestactices`
> 
> I did understand anything.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
Blocked on GitHub Actions CI job passing: https://github.com/github/opensource.guide/pull/1744/checks#step:7:38

**Review by Loscello — APPROVED** [review]

**Comment by zrzjohn** [comment]
> Need to fix the failing CI tests on this PR.

No GitHub API authentication could be found. Some fields may be missing or have incorrect data？

**Review by Azasyaid404 — APPROVED** [review]

**Review by Mohamad1-1994 — APPROVED** [review]

**Review by Mohamad1-1994 — APPROVED** [review]

**Review by agdgdh — APPROVED** [review]
Especially language bugs 

This problems are human common issue.

**Review by agdgdh — APPROVED** [review]

File: _articles/zh-hans/building-community.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ redirect_from: /zh-cn/building-community/
```
Comment (jomjamza) [inline _articles/zh-hans/building-community.md:27]: zrzjohn:master

**Review by jomjamza — APPROVED** [review]

**Comment by jomjamza** [comment]
zrzjohn:master

**Comment by jomjamza** [comment]
> Blocked on GitHub Actions CI job passing: https://github.com/github/opensource.guide/pull/1744/checks#step:7:38

**Comment by jomjamza** [comment]
> > Hi @zrzjohn - we can review this after you fix the errors in CI. For one thing, it looks like you auto-renamed `pr` to `Pull Request`, which resulted in some unintended changes, like renaming `best-practices` to `best-Pull Requestactices`
> 
> I don't know anything.

**Review by Anon79 — CHANGES_REQUESTED** [review]
Anon79(เจ้าของ)

**Review by Felipe1198 — CHANGES_REQUESTED** [review]
Osso

**Review by Yoshio123 — APPROVED** [review]

**Review by Yoshio123 — APPROVED** [review]

**Review by SaavyGirl — APPROVED** [review]

**Review by 041015778 — APPROVED** [review]

**Review by ashtont8 — APPROVED** [review]

**Review by 4264746 — CHANGES_REQUESTED** [review]
п

File: _articles/zh-hans/how-to-contribute.md [hunk 2]
```diff
@@ -45,109 +43,109 @@ redirect_from: /zh-cn/how-to-contribute/
...
 * 帮助社区成员寻找合适的技术会议，且帮助有实力的成员提交演讲的拟稿
 
 ### 是否偏向于设计？
 
 * 重新布置布局以提高项目的可用性
-* 进行用户研究以重新组织和完善项目的导航或菜单
+* 进行用户研究以重新计划和完善项目的导航或菜单
 * 整理一个风格指南，以帮助项目有一致的视觉设计
-* 创建t恤的艺术或一个新的标志，[就像 hapi.js 的贡献者那样](https://github.com/hapijs/contrib/issues/68)
+* 创建t恤的艺术或一个新的标志，[就像 hapi.js 的贡献者那样](https://github.com/hapijs/contrib/Issues/68)
```
Comment (WillemJiang) [inline _articles/zh-hans/how-to-contribute.md:89]: Please don't change the url, as the new link is broken. 
请保持原有的链接， 新修改的链接是不能访问的。

File: _articles/zh-hans/best-practices.md [hunk 3]
```diff
@@ -2,40 +2,38 @@
 lang: zh-hans
 title: 维护者最佳实践
 description: 身为开源的维护者，如何轻松驾驭项目？本指南从文档流程到有效利用社区来展开。
-class: best-practices
+class: best-Pull Requestactices
```
Comment (WillemJiang) [inline _articles/zh-hans/best-practices.md:5]: Could you explain why do you make this kind of change?

File: _articles/zh-hans/best-practices.md [hunk 4]
```diff
@@ -68,7 +66,7 @@ redirect_from: /zh-cn/best-practices/
 
 ### 保证交流是公开进行的
 
-不管是什么时候，保证你的交流是在公共的场所（就是大家都能看到的地方）。如果有人尝试和你私聊，哪怕是讨论一个新的需求或者功能，请礼貌的引导 Ta 到公共的交流场所，比如邮件列表或者 issue tracker。
+不管是什么时候，保证你的交流是在公共的场所（就是大家都能看到的地方）。如果有人尝试和你私聊，哪怕是讨论一个新的需求或者功能，请礼貌的引导 Ta 到公共的交流场所，比如邮件列表或者 Issue tracker。
```
Comment (WillemJiang) [inline _articles/zh-hans/best-practices.md:69]: Please don't replace issue with Issue globally, as it just broke the old links of github issue. 
请不要全文替换 issue， 这样会导致 github issue链接失效。

File: _articles/zh-hans/best-practices.md [hunk 5]
```diff
@@ -112,9 +110,9 @@ redirect_from: /zh-cn/best-practices/
 * **解释为什么他们的贡献不符合** 项目的需求范围，然后提供清楚的建议以供改善，如果你可以的话。和蔼一点，但同时也要讲原则。
 * **引用相关的文档，** 如果你有的话。如果你发现你反复收到你不想接受的贡献，把他们加到文档以免你重复叙述。
 
-你不需要用超过 1-2 两句话来回复。比如，当一个[celery](https://github.com/celery/celery/)的用户报告了一个window相关的错误，@berkerpeksag 是[这么](https://github.com/celery/celery/issues/3383)回复的
+你不需要用超过 1-2 两句话来回复。比如，当一个[celery](https://github.com/celery/celery/)的用户报告了一个window相关的错误，@berkerpeksag 是[这么](https://github.com/celery/celery/Issues/3383)回复的
 
-![celery screenshot](/assets/images/best-practices/celery.png)
+![celery screenshot](/assets/images/best-Pull Requestactices/celery.png)
```
Comment (WillemJiang) [inline _articles/zh-hans/best-practices.md:115]: I think it's just break the old link after the global replacement.

File: _articles/zh-hans/best-practices.md [hunk 6]
```diff
@@ -177,7 +175,7 @@ redirect_from: /zh-cn/best-practices/
 
 如果别人认同项目的发展方向，给他们提交的权限或者正式把项目所有权转移给他。如果有人 fork 了你的项目而且在保持活跃的维护中，考虑在你的原始的仓库放上这个 fork 版本的链接。如果大家都希望你的项目继续的话这不失为一种好办法。
 
-[@progruim](https://github.com/progrium) [发现](https://web.archive.org/web/20151204215958/https://progrium.com/blog/2015/12/04/leadership-guilt-and-pull-requests/) 由于它给他的项目[Dokku](https://github.com/dokku/dokku)写一个关于项目发展方向的文档，即使在它离开这个项目后他的那些目标仍然会被实现。
+[@Pull Requestogruim](https://github.com/Pull Requestogrium) [发现](https://web.archive.org/web/20151204215958/https://Pull Requestogrium.com/blog/2015/12/04/leadership-guilt-and-pull-requests/) 由于它给他的项目[Dokku](https://github.com/dokku/dokku)写一个关于项目发展方向的文档，即使在它离开这个项目后他的那些目标仍然会被实现。
```
Comment (WillemJiang) [inline _articles/zh-hans/best-practices.md:178]: It's just break lot of links.

**Review by WillemJiang — CHANGES_REQUESTED** [review]

**Pull request closed without merging**